flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/PartitionRequestNotifierTimeout.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.netty;
+
+import org.apache.flink.runtime.io.network.partition.PartitionRequestNotifier;
+
+/**
+ * Timeout message of given {@link PartitionRequestNotifier}.
+ */
+public class PartitionRequestNotifierTimeout {
```

Comment:
Why do we need this class? It seems just contains a `PartitionRequestNotifier`. An interface might be good enough?

BTW, I think we should also reconsider the structure and scope of `PartitionRequestNotifier`.

Reply:
Removed